### PR TITLE
Fix C and C++ Metadata Descriptions

### PR DESCRIPTION
**Pull request by Prince25**
### Description

As I was browsing through the docs, I noticed some metadata descriptions seemed auto-generated from the content in the body. Since it didn't make sense for some to have code blocks in the description, I did my best to paraphrase. 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Prince25** [comment]
There is more content that needs similar edits but I wanted to check if this type of change is valid before I continue.

**Review by SSwiniarski — APPROVED** [review]
@Prince25 This all looks good to me.

**Comment by Prince25** [comment]
Awesome. I will continue to make these changes to the rest of the content.

File: content/c/concepts/comments/comments.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Comments'
-Description: 'A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code. Single-line comments are created using two consecutive forward slashes. The compiler ignores any text after // on the same line. c // This line will denote a comment in C  Multi-line comments are created using / to begin the comment, and / to end the comment. The compiler ignores any text in between. c'
+Description: 'Comments are a piece of text within a program that are not executed. They are used to provide additional information to aid in understanding the code. In C, there are two types of comments: single and multi-line.'
```
Comment (yangc95) [inline content/c/concepts/comments/comments.md:3]:
```suggestion
Description: 'Comments are a piece of text within a program that are not executed. They are used to provide additional information to aid in understanding the code. In C, there are single and multi-line comments.'
```

File: content/c/concepts/user-input/user-input.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'User Input'
-Description: 'The scanf() function, which stands for “scan format", reads a formatted string from the standard input stream, which is usually the user keyboard. pseudo scanf("formatted string", &variable);  Here, the user can enter a value in the terminal, press Enter, and that number will get stored in the pinNumber variable. c scanf("%d", &pinNumber);'
+Description: 'For user input, the C language has the scanf function, which stands for scan format. It reads a formatted string from the standard input stream, which is usually the user keyboard.'
```
Comment (yangc95) [inline content/c/concepts/user-input/user-input.md:3]:
```suggestion
Description: 'In the C language, the scanf function stands for scan format and handles user input. It reads a formatted string from the standard input stream, which is usually the user keyboard.'
```

File: content/cpp/concepts/enum/enum.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Enums'
-Description: 'In C++, an enumeration (enum) is a user defined type where we specify a set of values for a variable and the variable can only take one out of a small set of possible values. The keyword enum is used to define an enumeration. cpp enum name {const1, const2, ...};  Heres an example: cpp'
+Description: 'In C++, an enumeration (enum) is a user defined type where we specify a set of values for a variable and the variable can only take one out of a small set of possible values.'
```
Comment (yangc95) [inline content/cpp/concepts/enum/enum.md:3]:
```suggestion
Description: 'In C++, an enumeration (enum) is a user defined type where a set of values is specified for a variable and the variable can only take one out of a small set of possible values.'
```

File: content/cpp/concepts/files/files.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Files'
-Description: 'Computers use file systems to store and retrieve data. Each file is an individual container of related information. The fstream library, short for "file stream", allows us to work with files in C++. To use the fstream library in the C++ program, include both the standard  and the  header files in the C++ program: cpp  There are three classes included in the fstream library, which are used to create, write or read files: - ofstream ("output file stream"): Create files and write to files. - ifstream ("input file stream"): Read from files.'
+Description: 'Computers use file systems to store and retrieve data. The fstream library, short for file stream, enables working with files in C++. The fstream library has three classes that are used to create, write or read files: ofstream, ifstream, and fstream.'
```
Comment (yangc95) [inline content/cpp/concepts/files/files.md:3]:
```suggestion
Description: 'Computers use file systems to store and retrieve data. The fstream library, short for file stream, enables working with files in C++. The fstream library has three classes that are used to create, write, and read files: ofstream, ifstream, and fstream.'
```

File: content/cpp/concepts/loops/loops.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Loops'
-Description: 'A loop can execute a statement or group of statements multiple times and the following is the general from of a loop statement in most programming languages. A while loop statement repeatedly executes the code block within as long as the condition is true. The moment the condition becomes false, the program will exit the loop. Note that the while loop might not ever run. If the condition is false initially, the code block will be skipped. cpp while (password != 1234) { std::cout > password;'
+Description: 'C++ has two types of loops: For and While. Loops execute the same group of statements multiple times until a stop condition is met.'
```
Comment (yangc95) [inline content/cpp/concepts/loops/loops.md:3]:
```suggestion
Description: 'C++ has two types of loops: For and While. Loops execute the same group of statements multiple times until a condition is no longer true.'
```

File: content/cpp/concepts/random/random.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Random'
-Description: 'C++ has a std::rand() function from cstdlib library that generates a random number. For example, if we add #include , we can use the std::rand() function: cpp std::cout << std::rand() << "\n"; std::cout << std::rand() << "\n"; std::cout << std::rand() << "\n"; '
+Description: 'C++ contains a function called rand from the cstdlib library that generates a random number.'
```
Comment (yangc95) [inline content/cpp/concepts/random/random.md:3]:
```suggestion
Description: 'C++ contains a function from the cstdlib library called rand that generates a random number.'
```

**Comment by yangc95** [comment]
@Prince25 Thanks for these metadata edits! Although they're not visible in the entries themselves, we do appreciate all appropriate enhancements to Docs 🙂

**Pull request merged**